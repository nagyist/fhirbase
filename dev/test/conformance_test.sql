--db:fhirb -e
SET escape_string_warning=off;
SET escape_string_warning=off;

--{{{
BEGIN;

SELECT assert_eq('version', (SELECT fhir_conformance('{"version":"version"}'))->>'version', 'version');

SELECT assert_eq('Patient',
   (fhir_profile(null::jsonb, 'Patient')#>>'{structure,0,element,0,path}'),
  'conformance');
ROLLBACK;

--}}}
