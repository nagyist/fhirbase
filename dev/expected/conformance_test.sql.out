SET escape_string_warning=off;
SET
SET escape_string_warning=off;
SET
BEGIN;
BEGIN
SELECT assert_eq('version', (SELECT fhir_conformance('{"version":"version"}'))->>'version', 'version');
 assert_eq  
------------
 OK version
(1 row)

SELECT assert_eq('Patient',
   (fhir_profile(null::jsonb, 'Patient')#>>'{structure,0,element,0,path}'),
  'conformance');
ROLLBACK;
ROLLBACK
